fvtest/rastest/CMakeLists.txt

```diff
@@ -0,0 +1,172 @@
+###############################################################################
+# Copyright (c) 2017, 2017 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at http://eclipse.org/legal/epl-2.0
+# or the Apache License, Version 2.0 which accompanies this distribution
+# and is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following Secondary
+# Licenses when the conditions for such availability set forth in the
+# Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+# version 2 with the GNU Classpath Exception [1] and GNU General Public
+# License, version 2 with the OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+###############################################################################
+
+add_tracegen(omr_test.tdf)
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
+if(CMAKE_C_COMPILER_ID STREQUAL MSVC)
+	if(CMAKE_VERSION VERSION_LESS 3.4)
+		message(WARN "CMake version 3.4 or higher is required for rastests to pass with MSVC")
```

Comment:
Why does windows require a newer CMake?

Reply:
`CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` was not added until 3.4

Reply:
And there is nothing we can do in older version to replicate the behaviour?  I really do not want to have one platform requiring such a new version compared to the others

Reply:
At some point I do plan on manually specifying exports for dll targets. But not anything I know of that can replicate the results trivially

Reply:
Actually since they are all agents with the same exports, I can whip up something without too much effort